src/uefistored.c

```diff
@@ -571,7 +580,7 @@ int main(int argc, char **argv)
 {
     xc_dominfo_t domain_info;
     shared_iopage_t *shared_iopage;
-    size_t vcpu_count = 1;
+    vcpu_count = 1;
```

Comment:
You can move the initialization directly where `vcpu_count` is declared in the global namespace.